src/main/scala/gestalt/Quote.scala

```diff
@@ -183,7 +183,7 @@ abstract class Quote(val t: Toolbox, val toolboxName: String) {
       // Infix is sugar-less
       // the List is constructed as Nil.::(c).::(b).::(a)
       // Not a :: b :: c :: Nil
```

Comment:
Nice, `Infix` works properly now!
The comment now is wrong, and should be removed.